docs/journalist.rst

```diff
@@ -519,44 +528,88 @@ by default, which may reveal sensitive information about your SecureDrop
 usage patterns (potentially including GPS coordinates) to anyone who gains access
 to the file.
 
-Encrypting and Moving Documents to Your Everyday Workstation
-------------------------------------------------------------
+Moving Documents to Your Everyday Workstation
+---------------------------------------------
+
+.. important::
+
+   As noted above, SecureDrop does not scan for or remove malware. If the file
+   you received contains malware targeting the operating system and applications
+   running on your everyday workstation, copying it in its original form carries
+   the risk of spreading malware to that computer. Make sure you understand the
+   risks, and consider other methods to export the document (e.g., print).
+
+If you must copy a file from your *Secure Viewing Station* to your everyday
+workstation in digital form, our :doc:`recommendation <set_up_transfer_and_export_device>`
+is that journalists are provided with an *Export Device*, typically a USB drive,
+which is encrypted using `VeraCrypt <https://www.veracrypt.fr/en/Home.html>`__.
+These instructions assume that you are following the recommended workflow.
+If you are unsure, ask your administrator.
```

Comment:
I think moving this a bit closer to the top, we want to ensure that admins in an org are aware and comfortable with journalists  opening those files on their everyday workstations.

Reply:
This document is the Journalist Guide, so I don't think we can assume that admins will ever read it. A fundamental challenge with having a single Journalist Guide is that it may include instructions that differ from the ones the organization is giving to its users.

In terms of alerting admins to the risk management aspect, I would suggest that we add some language to the overview section: https://docs.securedrop.org/en/release-0.14.0/overview.html